WordPress/Classes/ViewRelated/Plans/PlanPostPurchaseViewController.swift

```diff
@@ -335,9 +335,11 @@ enum PlanPostPurchasePageType: Int {
     case Themes
     
     static func pageTypesForPlan(plan: Plan) -> [PlanPostPurchasePageType] {
-        switch plan.slug {
-        case "premium": return [.PurchaseComplete, .Customize, .VideoPress]
-        case "business": return [.PurchaseComplete, .Customize, .VideoPress, .Themes]
+        switch plan.id {
+        // Premium
+        case 1003: return [.PurchaseComplete, .Customize, .VideoPress]
```

Comment:
I think we decided to leave them hardcoded for now... although thinking about it, we could quite easily make them dynamic without having to alter the API. We could simply select which screens to show based on the features that the purchased plan has.